.github/workflow-templates/bot-coin-family.yml

```diff
@@ -38,46 +38,15 @@ jobs:
       - name: Retrieving coin apps
         uses: actions/checkout@v3
         with:
+          ref: ledger-live-bot
           repository: LedgerHQ/coin-apps
           token: ${{ secrets.GH_PAT }}
           path: coin-apps
-      - uses: pnpm/action-setup@v2.0.1
-        with:
-          version: latest
-      - uses: actions/setup-node@v3
-        with:
-          node-version: 16.x
-          cache: pnpm
-          cache-dependency-path: "**/pnpm-lock.yaml"
-      - name: pull docker image
-        run: docker pull ghcr.io/ledgerhq/speculos
-      - name: install and build
-        run: |
-          pnpm i --filter="live-cli..." --filter="ledger-live" --frozen-lockfile
-          pnpm build:cli
-      - name: bot logic
-        env:
-          SEED: ${{ secrets.SEED3 }}
-          BOT_REPORT_FOLDER: botreport
-          VERBOSE_FILE: botreport/logs.txt
-          GITHUB_RUN_ID: ${{ github.run_id }}
-          GITHUB_WORKFLOW: ${{ github.workflow }}
-          BOT_FILTER_FAMILY: $coin
-        run: |
-          mkdir libs/ledger-live-common/botreport
-          COINAPPS=$PWD/coin-apps pnpm common ci-test-bot
+      - uses: ./tools/actions/composites/bot
+        id: bot
         timeout-minutes: 120
-      - name: submit bot report
-        uses: ledgerhq/ledger-live/tools/actions/submit-bot-report@develop
-        with:
-          path: libs/ledger-live-common/botreport/
-          githubSha: ${{github.sha}}
-          githubToken: ${{github.token}}
-          slackApiToken: ${{secrets.SLACK_API_TOKEN}}
-          slackChannel: ci-$coin-ll
-      - name: upload logs to github artifacts
-        if: always()
-        uses: actions/upload-artifact@v1
         with:
-          name: botreport
-          path: libs/ledger-live-common/botreport/
+          SLACK_API_TOKEN: ${{secrets.SLACK_API_TOKEN}}
+          SEED: ${{ secrets.SEED4 }}
+          SLACK_CHANNEL: ci-$COIN-ll
+          BOT_FILTER_FAMILY: $COIN
```

Comment:
I assume the values provided dynamically will override these?

Reply:
`$COIN` is a convention we take, github don't have this template actually, we just will have to manually replace $COIN, it make it easy to spot (i didn't change this, only uppercased it here)